js/react_native/android/build.gradle

```diff
@@ -136,4 +169,9 @@ dependencies {
   // Mobile build:
   // implementation "com.microsoft.onnxruntime:onnxruntime-mobile:latest.integration@aar"
   implementation "com.microsoft.onnxruntime:onnxruntime-android:latest.integration@aar"
+
+  // By default it will just include onnxruntime full aar package
+  if (getOnnxruntimePackageName() == "onnxruntime-ext") {
```

Comment:
nit: can store the result of `getOnnxruntimePackageName() == "onnxruntime-ext"` in a variable so we only need to run it once